.rspec

```diff
@@ -1 +1,2 @@
 --color
+--require spec_helper
```

Comment:
This change (+ the removals from the spec files) is so great. Thanks for doing it!